services/httpd/response_logger.go

```diff
@@ -104,15 +107,17 @@ func buildLogLine(l *responseLogger, r *http.Request, start time.Time) string {
 		"-",
 		detect(username, "-"),
 		fmt.Sprintf("[%s]", start.Format("02/Jan/2006:15:04:05 -0700")),
-		r.Method,
+		`"` + r.Method,
 		uri,
-		r.Proto,
+		r.Proto + `"`,
 		detect(strconv.Itoa(l.Status()), "-"),
 		strconv.Itoa(l.Size()),
-		detect(referer, "-"),
-		detect(userAgent, "-"),
+		`"` + detect(referer, "-") + `"`,
+		`"` + detect(userAgent, "-") + `"`,
 		r.Header.Get("Request-Id"),
-		fmt.Sprintf("%s", time.Since(start)),
+		// response time, report in microseconds because this is consistent
+		// with apache's %D parameter in mod_log_config
+		fmt.Sprintf("%d", time.Since(start).Nanoseconds()/1000),
```

Comment:
In implementing this for Kapacitor I noticed that we should probably change this to `strcnv.Itoa` to be consistent and probably faster. 

`strconv.Itoa(time.Since(start).Nanoseconds()/1000)`